persistence/mongodb-record/src/main/scala/net/liftweb/mongodb/record/field/DBRefField.scala

```diff
@@ -81,13 +81,11 @@ class DBRefField[OwnerType <: BsonRecord[OwnerType], RefType <: MongoRecord[RefT
   // assume string is json
   def setFromString(in: String): Box[DBRef] = {
     val dbo = JSON.parse(in).asInstanceOf[BasicDBObject]
-    MongoDB.use(ref.meta.connectionIdentifier) ( db => {
-      val id = dbo.get("$id").toString
-      ObjectId.isValid(id) match {
-        case true => Full(set(new DBRef(db, dbo.get("$ref").toString, new ObjectId(id))))
-        case false => Full(set(new DBRef(db, dbo.get("$ref").toString, id)))
-      }
-    })
+    val id = dbo.get("$id").toString
+    ObjectId.isValid(id) match {
+      case true => Full(set(new DBRef(dbo.get("$ref").toString, new ObjectId(id))))
+      case false => Full(set(new DBRef(dbo.get("$ref").toString, id)))
+    }
```

Comment:
So can this no longer fail? I see we're going to a `Full` box always, but we weren't before I guess?